src/screens/PluginStore/UploadPluginModal.tsx

```
@@ -105,196 +101,196 @@ const UploadPluginModal: React.FC<IUploadPluginModalProps> = ({
+          {t('uploadPluginModal.modalDescription')}
+        </p>
 
+        <div
+          className={`border-2 border-dashed p-5 text-center rounded ${
+            selectedFile ? 'border-primary bg-light' : 'border-secondary'
+          } ${styles.uploadArea}`}
+          onDragOver={handleDragOver}
+          onDrop={handleDrop}
+          onClick={() => fileInputRef.current?.click()}
+        >
           <input
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)